blog/2022/operator-crs.adoc

```diff
@@ -0,0 +1,29 @@
+:title: The future of Keycloak Operator CRs
+:date: 2022-09-01
+:publish: true
+:author: Václav Muzikář
+
+A while back, we have https://www.keycloak.org/2021/10/keycloak-x-update.html#_operator_and_containers[announced] a new
+Operator rewritten from scratch to provide the best experience for the Quarkus distribution. While the
+https://github.com/keycloak/keycloak-operator[legacy Operator] is now deprecated and
+https://www.keycloak.org/2022/03/releases.html[will reach EOL with Keycloak 20], the
+https://github.com/keycloak/keycloak/tree/main/operator[new one] is already available as
+a preview, see the https://www.keycloak.org/operator/installation[installation guide].
+
+One of the most common concerns around the new Operator is the current lack of the CRDs for managing Keycloak resources,
+such as realm, users and clients, in a cloud-native way. One of the key aspects of the new Operator will be redesign
+of managing these Keycloak resources via CRs and git-ops. This new approach will leverage
+the https://www.keycloak.org/2022/07/storage-map.html[new storage architecture] and future immutability options,
+making the CRs the declarative single source of truth. In comparison to the legacy Operator, this will bring high robustness,
+reliability, and predictability to the whole solution.
+
+The new storage is still work in progress and currently is lacking some features like the file store to persist
+data in a file instead of DB. Additionally, more features will have to be added to Keycloak, including but not limited to
+a completely new concept of immutability which will be used to ensure any data coming from the CRs (and subsequently from
+the file store) are read-only from the REST API, UI and other interfaces. All of this is critical to proper CRs implementation,
+hence the new Operator is currently missing the CRDs for managing Keycloak resources. The missing CRDs will be added once
+Keycloak has the necessary support for it, which is currently expected in Keycloak 21 (please note this is subject to
+change).
+
+In the meantime we have prepared a https://github.com/keycloak/keycloak-k8s-resources/tree/operators-tandem[few workarounds]
```

Comment:
Repository referred by this URL is empty (?)

Also would be good to start this paragraph with explanation of what workarounds could be expected so that the reader would know prior to following the link.

Reply:
Then I would suggest referring directly to README file and stating it:
```suggestion
We have prepared a few options to alleviate the situation with missing CRDs
in https://github.com/keycloak/keycloak-k8s-resources/tree/operators-tandem[this repository].
If you would like to try the new operator out, please see those options described in
in https://github.com/keycloak/keycloak-k8s-resources/blob/operators-tandem/README.md[README] file.
```
WDYT?

Reply:
Thanks, your wording seems better to me. Updated the PR.

The README is in the root of the branch, so when a user clicks the link to the branch itself, it opens the readme for them too. Therefore I wouldn't put a second link just to the README there.